.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -1,7 +1,7 @@
 name: Bug Report
 description: File a bug report
-title: "[Bug]: "
-labels: []
+title: ""
```

Comment:
The empty title seems to be invalid, but you can just remove it, as it's optional.

Reply:
What do you mean by it seems to be invalid?

Reply:
And calling it out now: I'm not entirely certain, that you can get way with not quoting the tag.